app/controllers/users_controller.rb

```diff
@@ -35,4 +48,12 @@ def check_user
 
     flash[:danger] = t "common.not_found"
   end
+
+  def correct_user
+    @user = User.find_by id: params[:id]
+    return if @user.current_user? current_user
+
+    flash[:danger] = t "Yon don't belong there"
```

Comment:
em dùng chữ thường và dấu gạch dưới để đặt tên biến I18n nhé, và tên này mang tính chất gợi nhớ thôi chứ ko phải câu văn đầy đủ thế kia.
ví dụ như:
```suggestion
    flash[:danger] = t "wrong_user"
```